AutocompleteClient/FW/API/Parser/Quizzes/QuizResultsResponseParser.swift

```diff
@@ -13,12 +13,40 @@ class QuizResultsResponseParser: AbstractQuizResultsResponseParser {
 
         do {
             let json = try JSONSerialization.jsonObject(with: quizResultsResponseData) as? JSONObject
-            let versionId = json?["version_id"] as? String ?? ""
+            
+            guard let response = json?["response"] as? JSONObject else {
+                throw CIOError(errorType: .invalidResponse)
+            }
+
+            let facetsObj: [JSONObject]? = response["facets"] as? [JSONObject]
+            let resultsObj: [JSONObject]? = response["results"] as? [JSONObject]
+            let sortOptionsObj: [JSONObject]? = response["sort_options"] as? [JSONObject]
+            let groupsObj = response["groups"] as? [JSONObject]
+
+            let facets: [CIOFilterFacet] = (facetsObj)?.compactMap { obj in return CIOFilterFacet(json: obj) } ?? []
+            let results: [CIOResult] = (resultsObj)?.compactMap { obj in return CIOResult(json: obj) } ?? []
+            let sortOptions: [CIOSortOption] = (sortOptionsObj)?.compactMap({ obj  in return CIOSortOption(json: obj) }) ?? []
+            let groups: [CIOFilterGroup] = groupsObj?.compactMap({ obj  in return CIOFilterGroup(json: obj) }) ?? []
+            let totalNumResults = response["total_num_results"] as? Int ?? 0
+            let resultSources: CIOResultSources? = CIOResultSources(json: response["result_sources"] as? JSONObject)
```

Comment:
I noticed that we don't currently include `ResultSources` for Quizzes in the .NET client or Java client (where we've made the `/finalize` => `/results` change). Should we add those there as well?